tools/geninterop/geninterop.py

```diff
@@ -244,42 +244,39 @@ def preprocess_python_headers():
 
 
 def gen_interop_head(writer):
-    defines = [
-        "PYTHON{0}{1}".format(PY_MAJOR, PY_MINOR)
-    ]
-
     if hasattr(sys, "abiflags"):
         if "d" in sys.abiflags:
             defines.append("PYTHON_WITH_PYDEBUG")
+            raise NotImplementedError
         if "m" in sys.abiflags:
             defines.append("PYTHON_WITH_PYMALLOC")
+            raise NotImplementedError
```

Comment:
This one has the exact same slots and is pretty widely used.

Reply:
The code in this method actually never passes any parameters to clang. It only used to affect the generated file by adding an extra compile-time condition.

My understanding is our current tools never actually run geninterop, and the generated files we have are all generated with none of the extra flags set.

Do we actually plan to support users regenerating interop files on their own? If yes, where is the manual (I could update it for all flags then along with this code)? If we do not, I can add a string to `NotImplementedError` with minimal details.

Reply:
`geninterop` is always run when in the `setup.py` we see that the respective file doesn't exist, yet. The `PYTHON_WITH_PYMALLOC` flag is quite common, in particular Anaconda uses it.

Reply:
Not sure I understand the doc correctly, but it looks like the set of supported flags went to 0 with 3.8

Reply:
@filmor this and the other comment should be addressed now.